### DISCUSSION TITLE
whereDate returning null data

### DISCUSSION THREAD

**Discussion by Aamirbashir** [jenssegers/laravel-mongodb#2336] [Q&A]
- Laravel-mongodb Version: 3.8
- PHP Version:  8.0.13
- Database Driver & Version: 8.0.13

### Description:
I am trying to get data by date, but I get a null value
### Steps to reproduce
Model.
    protected $table="categories";
    protected $connection="mongodb";
    protected $dates=['created_at','updated_at'];
   
Query:
 $category = category::whereDate('created_at','2022-01-06')->get();
Data in database
created_at: 2022-01-06T05:18:06.849+0000

### Expected behaviour
I am trying to get data by date non of the query are working on a date.

### Actual behaviour
I am getting Null data

<details><summary><b>Logs</b>:</summary>

</details>

**Comment by Aamirbashir** [comment]
any one can help me on this.

**Reply by Davpyu** [reply]
this might be too late, but can you test this? https://github.com/jenssegers/laravel-mongodb/pull/2376
